test/integration/provider/rti.go

```diff
@@ -17,16 +17,16 @@ type ResourcesTrackerImpl struct {
 
 func (r *ResourcesTrackerImpl) InitializeResourcesTracker(machineClass *v1alpha1.MachineClass, secretData map[string][]byte, clusterName string) error {
 
-	clusterTag := "tag:kubernetes.io/cluster/" + clusterName
-	clusterTagValue := "1"
+	integrationtestTag := "tag:kubernetes.io/role/integration-test"
+	integrationTestTagValue := "1"
```

Comment:
Make them global variables as they are used in both `InitializeResourcesTracker` and `probeResources`